AppCenter/AppCenter/Internals/Util/MSPerformSelectorUtil.h

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import <Foundation/Foundation.h>
+
+@interface MSPerformSelectorUtil : NSObject
+
++ (void)performSelectorOnMainThread:(NSObject *)source withSelector:(SEL)selector withObjects:(NSObject *)objects, ...;
```

Comment:
Can you move this to `MSUtility`?

Reply:
At first, we did it, but we had some strange issues when we tried to change NSObject to MSUtility inside this category (also I asked about this NSObject behavior in our chat):
https://github.com/microsoft/appcenter-sdk-apple/pull/1901/commits/d096abd12e8e29cc4ce99f6c7e9de7039e2b96d4#r355985378